src/service/templates/service.model.js

```diff
@@ -0,0 +1,4 @@
+
+module.exports = function(options) {
+  // instantiate and return model adapter of choice
+}
```

Comment:
this should be generated from model templates, however I'm going to wait until we have traction on that as a standalone generator, less details get lost in the cross over